From 8ac7db0122c4a7ca75108c098f40aca0ec95de3f Mon Sep 17 00:00:00 2001
From: Elizabeth Figura <zfigura@codeweavers.com>
Date: Tue, 10 Jun 2025 18:15:59 -0500
Subject: [PATCH 018/293] ir50_32: Do not handle a NULL input pointer in
 ICM_DECOMPRESS_GET_FORMAT.

Native will crash here. Get rid of code we don't need.
---
 dlls/ir50_32/ir50.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/dlls/ir50_32/ir50.c b/dlls/ir50_32/ir50.c
index ed57e135a2c..1fba354180d 100644
--- a/dlls/ir50_32/ir50.c
+++ b/dlls/ir50_32/ir50.c
@@ -120,9 +120,6 @@ IV50_DecompressGetFormat( LPBITMAPINFO in, LPBITMAPINFO out )
 
     TRACE("ICM_DECOMPRESS_GETFORMAT %p %p\n", in, out);
 
-    if ( !in )
-        return ICERR_BADPARAM;
-
     if (compare_fourcc(in->bmiHeader.biCompression, IV50_MAGIC))
         return ICERR_BADFORMAT;
 
-- 
2.43.0

